packages/process/process-monitor.pony

```diff
@@ -170,17 +171,20 @@ type ProcessError is
   | CapError
   )
 
+    
 actor ProcessMonitor
   """
   Forks and monitors a process. Notifies a client about STDOUT / STDERR events.
   """
   let _notifier: ProcessNotify
   var _child_pid: I32 = -1
 
+  var _stdin_event:  AsioEventID = AsioEvent.none()
   var _stdout_event: AsioEventID = AsioEvent.none()
   var _stderr_event: AsioEventID = AsioEvent.none()
 
   var _read_buf: Array[U8] iso = recover Array[U8].undefined(4096) end
+  embed _pending: List[(ByteSeq, USize)] = _pending.create()
```

Comment:
Seems like it might be better to use `Array` here and not depend on `collections`.

Unless you had some other reason for preferring `List`?